_docs-2/getting-started/features.md

```diff
@@ -418,3 +417,5 @@ beginning and end of the range are split, compacted, and then merged.
 [multivolume]: {% durl administration/multivolume %}
 [Iterators]: {% durl development/iterators %}
 [monitor]: {% durl administration/monitoring-metrics %}
+[spi]: TODO
```

Comment:
Were you thinking of linking the javadoc here?  I suppose the SPI javadoc is not there yet?

Reply:
Merging into next-release now. Will resolve the TODO with a proper link before publishing to the main site.